addons/point_of_sale/static/src/js/chrome.js

```diff
@@ -244,7 +244,7 @@ var DebugWidget = PosBaseWidget.extend({
         this.$('.button.delete_orders').click(function(){
             self.gui.show_popup('confirm',{
                 'title': _t('Delete Paid Orders ?'),
-                'body':  _t('This operation will permanently destroy all paid orders from the local storage. You will lose all the data. This operation cannot be undone.'),
+                'body':  _t('This operation will permanently destroy all paid orders from the browser cache. You will lose all the data. This operation cannot be undone.'),
```

Comment:
Is it just me, or is 'You will lose all the data' weird? I would remove this phrase entirely